tsl/src/compression/compress_utils.c

```diff
@@ -124,58 +122,6 @@ compression_chunk_size_catalog_insert(int32 src_chunk_id, ChunkSize *src_size,
 	table_close(rel, RowExclusiveLock);
 }
 
-static void
-chunk_dml_blocker_trigger_add(Oid relid)
```

Comment:
Don't we need to block UPDATEs (and DELETES?) to compressed chunks still?

Reply:
This specific trigger was only used to block INSERTs. UPDATE/DELETE is still blocked but blocking happens in custom node. We needed the INSERT trigger here to also cover the COPY path